### PR TITLE
GUVNOR-2365: Repository remains in repositories list after it is deleted

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-project-explorer/kie-wb-common-project-explorer-client/src/main/java/org/kie/workbench/common/screens/explorer/client/widgets/ActiveContextManager.java [hunk 1]
```diff
@@ -276,7 +276,7 @@ public void onRepositoryRemovedEvent( @Observes final RepositoryRemovedEvent eve
         if ( !view.isVisible() ) {
             return;
         }
-        if ( activeContextItems.getActiveRepository().equals( event.getRepository() ) ) {
+        if ( event.getRepository().equals( activeContextItems.getActiveRepository() ) ) {
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-project-explorer/kie-wb-common-project-explorer-client/src/main/java/org/kie/workbench/common/screens/explorer/client/widgets/ActiveContextManager.java]: I assume ```activeContextItems.getActiveRepository()``` could have returned ```null``` (which seems plausible if no Repository had been selected in the Authoring Perspective before trying to delete a Repository).

**Comment by manstis** [comment]
A test (no matter how trivial) would be nice!

File: kie-wb-common-screens/kie-wb-common-project-explorer/kie-wb-common-project-explorer-client/src/main/java/org/kie/workbench/common/screens/explorer/client/widgets/ActiveContextManager.java (hunk 1 above)
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-project-explorer/kie-wb-common-project-explorer-client/src/main/java/org/kie/workbench/common/screens/explorer/client/widgets/ActiveContextManager.java]: @manstis Yes, it was that. That was why the problem seemed random.

**Comment by paulovmr** [comment]
@manstis Ok, you are right, I will make one. Thanks!

**Comment by paulovmr** [comment]
@manstis The tests and comment have been made.

**Comment by manstis** [comment]
+1 to merge. Thanks!

**Pull request merged**